diff --git a/node_modules/@react-native-google-signin/google-signin/lib/module/index.js b/node_modules/@react-native-google-signin/google-signin/lib/module/index.js
index 8a453da..976a937 100644
--- a/node_modules/@react-native-google-signin/google-signin/lib/module/index.js
+++ b/node_modules/@react-native-google-signin/google-signin/lib/module/index.js
@@ -1,8 +1,8 @@
 "use strict";
 
-export { GoogleSignin } from './signIn/GoogleSignin';
-export { statusCodes } from './errors/errorCodes';
-export { GoogleSigninButton } from './buttons/GoogleSigninButton';
+export { GoogleSignin } from './signIn/GoogleSignin.js';
+export { statusCodes } from './errors/errorCodes.js';
+export { GoogleSigninButton } from './buttons/GoogleSigninButton.js';
 export * from "./types.js";
 export * from "./functions.js";
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/@react-native-google-signin/google-signin/lib/module/signIn/GoogleSignin.js b/node_modules/@react-native-google-signin/google-signin/lib/module/signIn/GoogleSignin.js
index 7127c15..5d7388a 100644
--- a/node_modules/@react-native-google-signin/google-signin/lib/module/signIn/GoogleSignin.js
+++ b/node_modules/@react-native-google-signin/google-signin/lib/module/signIn/GoogleSignin.js
@@ -2,7 +2,7 @@
 
 import { Platform } from 'react-native';
 import { NativeModule } from "../spec/NativeGoogleSignin.js";
-import { ios_only_SCOPES_ALREADY_GRANTED, SIGN_IN_REQUIRED_CODE } from '../errors/errorCodes';
+import { ios_only_SCOPES_ALREADY_GRANTED, SIGN_IN_REQUIRED_CODE } from '../errors/errorCodes.js';
 import { noSavedCredentialFoundResult } from "../constants.js";
 import { translateCancellationError } from "../translateNativeRejection.js";
 import { isErrorWithCode } from "../functions.js";
diff --git a/node_modules/@react-native-google-signin/google-signin/lib/module/translateNativeRejection.js b/node_modules/@react-native-google-signin/google-signin/lib/module/translateNativeRejection.js
index 08f9cd5..9bae825 100644
--- a/node_modules/@react-native-google-signin/google-signin/lib/module/translateNativeRejection.js
+++ b/node_modules/@react-native-google-signin/google-signin/lib/module/translateNativeRejection.js
@@ -2,7 +2,7 @@
 
 import { isErrorWithCode } from "./functions.js";
 import { cancelledResult } from "./constants.js";
-import { SIGN_IN_CANCELLED_CODE } from './errors/errorCodes';
+import { SIGN_IN_CANCELLED_CODE } from './errors/errorCodes.js';
 
 /**
  * Since the introduction of a new JS api, the native rejections need to be processed in JS layer.
